packages/apps/dashboard/server/src/common/constants/chains.ts

```diff
@@ -0,0 +1,22 @@
+import { ChainId as SdkChainId } from '@human-protocol/sdk';
+
+import Environment from '../utils/environment';
+
+export enum ProductionChainId {
+  POLYGON_MAINNET = SdkChainId.POLYGON,
+  BSC_MAINNET = SdkChainId.BSC_MAINNET,
+  ETHEREUM = SdkChainId.MAINNET,
+}
+
+export enum DevelopmentChainId {
+  POLYGON_AMOY = SdkChainId.POLYGON_AMOY,
+  BSC_TESTNET = SdkChainId.BSC_TESTNET,
+  SEPOLIA = SdkChainId.SEPOLIA,
+  LOCALHOST = SdkChainId.LOCALHOST,
+}
```

Comment:
```suggestion
export const ProductionChainId = {
  POLYGON_MAINNET: SdkChainId.POLYGON,
  BSC_MAINNET: SdkChainId.BSC_MAINNET,
  ETHEREUM: SdkChainId.MAINNET,
} as const satisfies Record<string, SdkChainId>;

export type ProductionChainId =
  (typeof ProductionChainId)[keyof typeof ProductionChainId];

export const DevelopmentChainId = {
  POLYGON_AMOY: SdkChainId.POLYGON_AMOY,
  BSC_TESTNET: SdkChainId.BSC_TESTNET,
  SEPOLIA: SdkChainId.SEPOLIA,
  LOCALHOST: SdkChainId.LOCALHOST,
} as const satisfies Record<string, SdkChainId>;

export type DevelopmentChainId =
  (typeof DevelopmentChainId)[keyof typeof DevelopmentChainId];
```

We can define objects as const instead of enum. This way it uses SdkChainId internally, so we don't have to specify `chainId as number` on each SDK call